SourceX/display.cpp

```diff
@@ -116,10 +121,24 @@ void CalculatePreferdWindowSize(int &width, int &height, bool useIntegerScaling)
 
 bool SpawnWindow(const char *lpWindowName)
 {
+
+#ifdef __vita__
+scePowerSetArmClockFrequency(444);
+#endif
+#if SDL_VERSION_ATLEAST(2,0,6)
+	SDL_SetHint(SDL_HINT_TOUCH_MOUSE_EVENTS, "0");
+#endif
 	if (SDL_Init(SDL_INIT_EVERYTHING & ~SDL_INIT_HAPTIC) <= -1) {
 		ErrSdl();
 	}
 
+	char mapping[1024];
+	memset(mapping, 0, 1024);
+	getIniValue("controls","sdl2_controller_mapping", mapping, 1024);
+	if (mapping[0] != '\0') {
+		SDL_GameControllerAddMapping(mapping);

```

Comment:
Need to hide this from SDL1:
```suggestion
#ifndef USE_SDL1
		SDL_GameControllerAddMapping(mapping);
#endif
```